task/buildah/0.2/buildah.yaml

```diff
@@ -303,6 +303,12 @@ spec:
 
       BUILDAH_ARGS+=("${BUILD_ARG_FLAGS[@]}")
 
+      if [ "${PRIVILEGED_NESTED}" == "true" ]; then
+        BUILDAH_ARGS+=("--security-opt=label=disable")
+        BUILDAH_ARGS+=("--cap-add=all")
+        BUILDAH_ARGS+=("--device=/dev/fuse")
+      fi
```

Comment:
The scripts sets `set -u`, so this will always fail because of the undefined variable.

Could you also add a comment that this code is only here because it's needed in the buildah-remote task?

Reply:
And there is another related issue, when I dropped PRIVILEGED_NESTED param in buildah.yaml, task-generator/remote/main.go won't add `-v PRIVILEGED_NESTED="$PRIVILEGED_NESTED"` to the `podman run` command runs over ssh, then the inner scripts/script-build.sh won't have $PRIVILEGED_NESTED set.

How about we add param PRIVILEGED_NESTED back to all buildah tasks to address all these problems, literally it have privilege mode enabled for local buildah.yaml because we add option `--security-opt=label=disable --cap-add=all` to `buildah build`, I just don't have a real world use case to verify it works for local build, but for bootc image we only use remote task for multi-arch image building,

Reply:
Added `PRIVILEGED_NESTED` back and removed "remote" keyword in description.
Bootc image building test is PASS.

Reply:
So we just don't know if it works in the non-remote task. That's fine ¯\\\_(ツ)\_/¯